.changelog/unreleased/improvements/ibc-relayer/4143-use-connection-params-in-validate-params.md

```diff
@@ -0,0 +1,3 @@
+- Use the gRPC query `ConnectionParams` to retrieve `maxExpectedTimePerBlock`
+  and verify the configured `max_block_time` instead of using the `/genesis`
+  endpoint. ([\#4143](https://github.com/informalsystems/hermes/issues/4143))
```

Comment:
```suggestion
- Use the `ibc.core.connection.v1.ConnectionParams` gRPC query to retrieve  `maxExpectedTimePerBlock` 
  and check it against the configured `max_block_time` instead of using the `/genesis`
  endpoint. ([\#4143](https://github.com/informalsystems/hermes/issues/4143))
```